### PR TITLE
Optimize es queries

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

File: guillotina_cms/__init__.py [hunk 1]
```diff
@@ -158,3 +164,11 @@ def includeme(root):
     configure.scan('guillotina_cms.install')
     configure.scan('guillotina_cms.subscribers')
     configure.scan('guillotina_cms.tiles')
+
+    # add store true to more guillotina indexes
+    for name, utility in get_utilities_for(IResourceFactory):
+        if not get_dotted_name(utility._callable).startswith('guillotina.'):
+            continue
+        for field_name, catalog_info in get_index_fields(name).items():
+            if field_name in ('creation_date', 'modification_date', 'tags'):
```
Comment (bloodbare) [inline guillotina_cms/__init__.py]: Maybe we could add this to guillotina ?

Comment (vangheem) [reply guillotina_cms/__init__.py]: we can do it there as well

Comment (bloodbare) [reply guillotina_cms/__init__.py]: On g_es

Comment (vangheem) [reply guillotina_cms/__init__.py]: done.

**Pull request merged**